src/core_modules/capture-core/components/Pages/ViewEvent/epics/getCategoriesDataFromEvent.js

```diff
@@ -15,7 +15,7 @@ function getCategoryOptionsAsync(optionIds: string) {
 
 
 export async function getCategoriesDataFromEventAsync(event: CaptureClientEvent): Promise<?Array<Object>> {
-    const optionIdsFromEvent = event.attributeCategoryOptions;
+    const optionIdsFromEvent = event.attributeCategoryOptions && event.attributeCategoryOptions.replace(/;/g, ',');
```

Comment:
Hey @EirikHaugstulen, 

You can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining). Just like you did in the PR for v36 https://github.com/dhis2/capture-app/pull/1882

Reply:
Hey @simonadomnisoru,
Flow config, or perhaps the specific version of Flow used in the v35 branch, marks optional chaining as an experimental feature and will fail the test. You can check the error message in the first commit in this PR.
Do you want me to check if I can enable optional chaining in the whole v35-branch, or should we leave it as it is?
Thanks!

Reply:
Good to know. Let's leave it like this. Thanks!